### PR TITLE
improve: Jetpack compose sample app (SDKCF-4860)

### PR DESCRIPTION
**Pull request by maureenorea-clores**
# Description
Compose sample app to verify IAM SDK support

## Links
SDKCF-4860

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys
- [ ] I checked that all possible internal and platform/library exceptions are gracefully handled and **will not** crash the host app
- [ ] I checked for open & known issues (especially crashes) for any newly added platform APIs or libraries
- [ ] I tested non-trivial changes on a real device
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: test-compose/src/androidTest/java/com/rakuten/tech/mobile/test_compose/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.rakuten.tech.mobile.test_compose
```
Comment (vincentdelmundo) [inline test-compose/src/androidTest/java/com/rakuten/tech/mobile/test_compose/ExampleInstrumentedTest.kt:1]: This file can be removed.

File: test-compose/src/test/java/com/rakuten/tech/mobile/test_compose/ExampleUnitTest.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.rakuten.tech.mobile.test_compose
```
Comment (vincentdelmundo) [inline test-compose/src/test/java/com/rakuten/tech/mobile/test_compose/ExampleUnitTest.kt:1]: This can be removed

**Review by vincentdelmundo — CHANGES_REQUESTED** [review]
Looks good. Just need to remove unnecessary files.

File: test-compose/src/androidTest/java/com/rakuten/tech/mobile/test_compose/ExampleInstrumentedTest.kt (hunk 1 above)
Comment (maureenorea-clores) [reply test-compose/src/androidTest/java/com/rakuten/tech/mobile/test_compose/ExampleInstrumentedTest.kt]: Removed

File: test-compose/src/test/java/com/rakuten/tech/mobile/test_compose/ExampleUnitTest.kt (hunk 2 above)
Comment (maureenorea-clores) [reply test-compose/src/test/java/com/rakuten/tech/mobile/test_compose/ExampleUnitTest.kt]: Removed

**Review by vincentdelmundo — APPROVED** [review]

**Pull request merged**